### PR TITLE
pushing data to code speed for pvc_creationa and deletation cases

### PR DESCRIPTION
**Pull request by SvetlanaRedHat**
Signed-off-by: savetisy <savetisy@redhat.com>

### CODE REVIEW

**Comment by SvetlanaRedHat** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/2389/

File: tests/e2e/performance/test_pvc_creation_performance.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 from tests import helpers
 from ocs_ci.ocs import defaults, constants
 
-
+from ocs_ci.utility.performance_dashboard import push_to_pvc_time_dashboard
```
Comment (AviadP) [inline tests/e2e/performance/test_pvc_creation_performance.py:16]: plz add blank line between imports and log object creation

**Comment by keesturam** [comment]
Please make the change Aviad suggested and CI issues. otherwise, looks good to me.

File: tests/e2e/performance/test_pvc_creation_performance.py (hunk 1 above)
Comment (SvetlanaRedHat) [reply tests/e2e/performance/test_pvc_creation_performance.py:16]: fixed

**Comment by SvetlanaRedHat** [comment]
@keesturam  was already fixed

File: tests/e2e/performance/test_pvc_creation_performance.py (hunk 1 above)
Comment (SvetlanaRedHat) [reply tests/e2e/performance/test_pvc_creation_performance.py:16]: fixed

**Comment by vasukulkarni** [comment]
Conflict, needs rebase

**Comment by vasukulkarni** [comment]
There are some flake8 issues that need fix.

**Comment by SvetlanaRedHat** [comment]
clsoing this

**Pull request closed without merging**